pom.xml

```diff
@@ -16,8 +16,9 @@
 
     <properties>
         <version.opentelemetry>1.29.0</version.opentelemetry>
-        <version.microprofile.opentelemetry>1.0</version.microprofile.opentelemetry>
+        <version.microprofile.opentelemetry>1.2-SNAPSHOT</version.microprofile.opentelemetry>
```

Comment:
Should this be 1.1-RC2 (iirc)

Reply:
It it the local SNAPSHOT I'm using to test because of https://github.com/eclipse/microprofile-telemetry/pull/129

Reply:
That makes sense. Thanks. :)